EIPS/eip-7777.md

```diff
@@ -0,0 +1,341 @@
+---
+eip: 7777
+title: ERC20Spendable
+description: An extension of the ERC20 standard, allowing ERC20 token transfers to trigger contract events and receive returned values.
+author: Omnus Sunmo (@omnus)
+discussions-to: https://ethereum-magicians.org/t/eip-pending-erc20spendable/
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-08-17
+requires: 20
+---
+
+## Abstract
+
+An extension of ERC20 that allows user's to initiate a transfer of tokens that:
```

Comment:
```suggestion
An extension of ERC-20 that allows user's to initiate a transfer of tokens that:
```

Reply:
Ooh, great catch, thank you! I forgot the hyphen was back in....